docs/experiments/interpreting-experiments/lift-estimates-and-confidence-intervals/statistical-nitty-gritty.md

```diff
@@ -0,0 +1,560 @@
+---
+sidebar_position: 2
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+
+import NoBreak from '@site/src/components/NoBreak'
+import Term from '@site/src/components/glossary/Term'
+
+# Statistical nitty-gritty
+
+The fixed-sample, sequential, and Bayesian methods all start from the same
+place: estimating the <NoBreak>**mean** ($\mu$)</NoBreak>
+and <NoBreak>**variance** ($\sigma^2$)</NoBreak> of the lift;
+the former is an estimate of the size of the treatment effect (as a percentage
+of the baseline metric value), and the latter is an indication of how reliable
+that estimate is for predicting what will happen if you ship the treatment.
+
+Once we have an estimate of the observed lift (both its mean and variance), we
+can use that to construct a confidence interval that describes the plausible
+values for the true lift.
+
+## Estimating lift
+
+First, some notation:
+
+* For a given metric, we observe $y_i$ for each subject $i$ in the control group
+  of size $n_C$, and $y_j$ for each subject $j$ in the treatment group of size
+  $n_T$. The set of all observations in the control group is $\bold{Y}_C$, and
+  for the treatment group it is $\bold{Y}_T$.
+
+* The *true* population mean (which we don't know!) of the metric among the control group
+  is $\mu_C$, and among the treatment group it is $\mu_T$. Similarly, the *true*
+  (unobserved) population variance is $\sigma^2_C$ and $\sigma^2_T$
+
+* $m_C$ and $m_T$ are the averages of $\bold{Y}_C$ and $\bold{Y}_T$ across all
+  subjects in the control group and treatment group, respectively:
+  
+  $$
+  \begin{equation}
+  \begin{split}
+  m_C &= \frac{\sum_{i=1}^{n_C}{y_i}}{n_C}
+  \\[1.2em]
+  m_T &= \frac{\sum_{j=1}^{n_T}{y_j}}{n_T}
+  \end{split}
+  \end{equation}
+  $$
+
+* $s^2_C$ and $s^2_T$ are the *sample* variances of $\bold{Y}_C$ and $\bold{Y}_T$
+  for the control group and treatment group, respectively:
+
+  $$
+  \begin{equation}
+  \begin{split}
+  s^2_C &= \frac{n_C}{n_C - 1}\left(\frac{\sum_{i=1}^{n_C}{y_i^2}}{n_C} - m_C^2\right)
+  \\[1.2em]
+  s^2_T &= \frac{n_T}{n_T - 1}\left(\frac{\sum_{j=1}^{n_T}{y_j^2}}{n_T} - m_T^2\right)
+  \end{split}
+  \end{equation}
+  $$
+
+Regardless of the distribution of $\bold{Y}_C$ and $\bold{Y}_T$, the
+[Central Limit Theorem (CLT)](https://en.wikipedia.org/wiki/Central_limit_theorem)
+says that, when $n_C$ and $n_T$ are sufficiently large, the *mean* of
+$\bold{Y}_C$ and $\bold{Y}_T$ are each normally distributed;
+that is:
+
+$$
+\begin{equation}
+\begin{align*}
+\lim_{n_C \to \infty}m_C \sim&~~ \mathcal{N}(\mu_C; \frac{\sigma^2_C}{n_C})
+\\[1em]
+\lim_{n_T \to \infty}m_T \sim&~~ \mathcal{N}(\mu_T; \frac{\sigma^2_T}{n_T})
+\end{align*}
+\end{equation}
+$$
+
+:::info Ratio metrics
```

Comment:
@schmit I won't block merge on this, but when you get a chance would love your 👀 to make sure I didn't mess anything up in translation from your version. I would really love a good reference for being able to do this normal ratio magic, where we approximate away the cauchy/t even though num. and denom. are correlated here